### PR TITLE
add new joystick2 and  the toggle camera perspective button

### PR DESCRIPTION
**Pull request by tulingcheng86**

### CODE REVIEW

File: webui/.vs/ProjectSettings.json
Comment (a-wing) [inline webui/.vs/ProjectSettings.json:1]: Git Ignore `.gitignore`

File: unity/Assets/joyrtc.cs [hunk 1]
```diff
@@ -1,174 +1,209 @@
-using System.Collections;
```
Comment (a-wing) [inline unity/Assets/joyrtc.cs:1]: Maybe this file have some problem

**Review by a-wing — CHANGES_REQUESTED** [review]
Need Remove `.vs` directory

File: webui/.vs/ProjectSettings.json
Comment (tulingcheng86) [reply webui/.vs/ProjectSettings.json:1]: ok

File: unity/Assets/joyrtc.cs (hunk 1 above)
Comment (tulingcheng86) [reply unity/Assets/joyrtc.cs:1]: what problem

Comment (a-wing) [reply unity/Assets/joyrtc.cs:1]: Why this file every line show change?

Comment (tulingcheng86) [reply unity/Assets/joyrtc.cs:1]: 我一开始git push 失败，后来先 git pull 下来，用vs来修改冲突，vs打开joyrtc.cs显示换行符什么错误，vs说可以一键修改，修改冲突后git push ，可能是这里导致的

Comment (tulingcheng86) [reply unity/Assets/joyrtc.cs:1]: 115行-140行修改了，20行-24行修改

Comment (a-wing) [reply unity/Assets/joyrtc.cs:1]: https://learn.microsoft.com/zh-cn/visualstudio/ide/encodings-and-line-breaks?view=vs-2022

Comment (a-wing) [reply unity/Assets/joyrtc.cs:1]: https://stackoverflow.com/questions/1967370/git-replacing-lf-with-crlf

File: unity/Assets/joyrtc.cs [hunk 2]
```diff
@@ -14,11 +17,17 @@ public class MyObject
...
+{
+    public MyObject joystick1;
+    public MyObject joystick2;
+}
 
 public class joyrtc : MonoBehaviour {
 #pragma warning disable 0649
 	[SerializeField] private Camera cam;
     [SerializeField] private GameObject cube; // 添加一个立方体游戏对象
+    private bool enableCameraModeToggle = false;
```
Comment (a-wing) [inline unity/Assets/joyrtc.cs:30]: Maybe this line should `#pragma warnin` outside ?

**Pull request merged**